src/pinwheel/pinwheel.style.ts

```diff
@@ -17,6 +17,9 @@ export default css`
   }
 
   .pinwheel > div {
+    display: flex;
+    align-items: center;
+    justify-content: center;
     padding: var(--tap-pinwheel-spacing-0, var(--tap-sys-spacing-0)) var(--tap-pinwheel-spacing-6, var(--tap-sys-spacing-6));
```

Comment:
same comment here. I think the numbers are for general and global tokens and here there is no need to name our variables.